internal/venice-test-common/src/integrationtest/java/com/linkedin/venice/integration/utils/VeniceClusterWrapper.java

```diff
@@ -554,6 +572,7 @@ public VeniceServerWrapper addVeniceServer(boolean enableAllowlist, boolean enab
     featureProperties.setProperty(SERVER_ENABLE_SERVER_ALLOW_LIST, Boolean.toString(enableAllowlist));
     featureProperties.setProperty(SERVER_IS_AUTO_JOIN, Boolean.toString(enableAutoJoinAllowList));
     VeniceServerWrapper veniceServerWrapper = ServiceFactory.getVeniceServer(
+        "",
```

Comment:
Let's use `coloName` like in the `addVeniceRouter` function. Same for other `addVeniceServer` functions below

Reply:
Today, we don't have a use case to add some dedicate Router/Server to a multiple colo setup, since the colo name is implicit.
The only known use case of this function is in a single cluster/colo setup.
And for the single colo setup, colo name is not necessary.

Reply:
Just got what you meant and I fixed them in the latest commit.